### DISCUSSION TITLE
website copyright - replace current year with <?= date("Y"); ?>

### DISCUSSION THREAD

**Discussion by nocturnalguru** [pulsar-edit/.github#265] [Documentation & Website]
I'd like to recommend that the footer of the pulsar website be updated so that the current year is displayed so the whole site looks like it's being maintained. 

Can someone replace the year 2023 in the footer copyright with <?= date("Y"); ?> so it automatically updates every Jan 1st.  Unless you know of a better way.

I love this project, but it looks dated..  and we are 4 months into the year.

**Comment by confused-Techie** [comment]
Thanks a ton for the recommendation. We will get to this when we can. If you want to try updating it yourself feel free to submit a [PR](https://github.com/pulsar-edit/pulsar-edit.github.io).
